nifi-nar-bundles/nifi-phoenix-bundle/nifi-phoenix-service/src/test/java/org/apache/nifi/dbcp/phoenix/TTTestStandardPhoenixDBCPService.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.dbcp.phoenix;
+
+import org.apache.nifi.reporting.InitializationException;
+import org.apache.nifi.util.TestRunner;
+import org.apache.nifi.util.TestRunners;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TTTestStandardPhoenixDBCPService {
```

Comment:
Is the triple leading T in the name `TTTestStandardPhoenixDBCPService` serves to purpose to _not_ recognise this as a test class (as the single test method seems to contain environment-specific configuration)?

Should we leave out or modify this class in that case?